app/pktgen-port-cfg.c

```diff
@@ -312,6 +312,11 @@ pktgen_config_ports(void)
 
         pktgen.mem_used = 0;
 
+        if ((ret = rte_eth_dev_set_mtu(pid, pktgen.eth_mtu)) < 0) {
+            pktgen_log_panic("Cannot set MTU %u on port %u, (%d)%s", pktgen.eth_mtu, pid,
+                             -ret, rte_strerror(-ret));
+        }
```

Comment:
Can you please remove the {} from around this single line statement. The coding style is suppose to be no {} around single line statements like for, while, if, else, ..